hub/core/dataset/dataset.py

```diff
@@ -1343,6 +1343,70 @@ def _set_read_only(self, value: bool, err: bool):
     def read_only(self, value: bool):
         self._set_read_only(value, True)
 
+    def cleanlab(
```

Comment:
I've spent a few hours thinking about the name but can't think of any better one 😅. I think `cleanlab` should be there as we're using a third-party library. Maybe `clean_labels_with_cleanlab`?